recipes/r-idm/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set version = '1.8.3' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-idm
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/idm_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/idm/idm_{{ version }}.tar.gz
+  sha256: 47b9b088b4caa526de729bad73496ef722dfb11637341fc74e730f94b0174c11
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-animation
+    - r-ca
+    - r-corpcor
+    - r-ggplot2
+    - r-ggrepel
+  run:
+    - r-base
+    - r-animation
+    - r-ca
+    - r-corpcor
+    - r-ggplot2
+    - r-ggrepel
+
```

`DESCRIPTION` states it requires `imagemagick`, so let's include for consistency:

```suggestion
    - imagemagick

```